README.md

```diff
@@ -186,91 +91,75 @@ kg = KnowledgeGraph(
     name="kg_name",
     model_config=KnowledgeGraphModelConfig.with_model(model),
     ontology=ontology,
+    host="127.0.0.1",
+    port=6379,
+    # username=falkor_username, # Not required for on-premises use.
+    # password=falkor_password
 )
 
 kg.process_sources(sources)
 ```
 
-You can update the KG at any time by processing more sources with the `process_sources` method.
-
-#### Configurable Prompts
-When creating your Knowledge Graph (KG) agent, you can customize the prompts to tailor its behavior. This step is optional but can enhance functionality.
-
-There are five types of prompts:
-
-1. **`cypher_system_instruction`**  
-   - System instructions for the Cypher generation step.  
-   - **Note:** Ensure your prompt includes `{ontology}`.
-
-2. **`qa_system_instruction`**  
-   - System instructions for the Q&A step.
-
-3. **`cypher_gen_prompt`**  
-   - The prompt used during the Cypher generation step.  
-   - **Note:** Include `{question}` in your prompt.
-
-4. **`cypher_gen_prompt_history`**  
-   - The prompt for Cypher generation when history needs to be considered.  
-   - **Note:** Include `{question}` and `{last_answer}` in your prompt.
-
-5. **`qa_prompt`**  
-   - The prompt used during the Q&A step.  
-   - **Note:** Include `{question}`, `{context}`, and `{cypher}` in your prompt.
-
-Here’s an example configuration:
-
-```python
-kg = KnowledgeGraph(
-    name="kg_name",
-    model_config=KnowledgeGraphModelConfig.with_model(model),
-    ontology=ontology,
-    cypher_system_instruction=cypher_system_instruction,
-    qa_system_instruction=qa_system_instruction,
-    cypher_gen_prompt=cypher_gen_prompt,
-    cypher_gen_prompt_history=cypher_gen_prompt_history,
-    qa_prompt=qa_prompt
-)
-```
-
-### Graph RAG
+### Step 3: Query your Graph RAG
 
 At this point, you have a Knowledge Graph that can be queried using this SDK. Use the method `chat_session` for start a conversation.
 
 ```python
-
-# Conversation.
+# Conversation
 chat = kg.chat_session()
-response = chat.send_message("Who is Salsa Boy?")
+response = chat.send_message("Who is the director of the movie The Matrix?")
 print(response)
-response = chat.send_message("Tell me about one of his fights?")
+response = chat.send_message("How this director connected to Keanu Reeves?")
 print(response)
 ```
 
-## Multi Agent - Orchestrator
+## Next Steps
+With these 3 steps now completed, you're ready to interact and query your knowledge graph.  Here are suggestions for use cases:
+<p align="left">
+  <img alt="GraphRAG-SDK Use Cases Banner from FalkorDB" src="images/use-cases.png" width="800">
+</p>
+
+**Need help with your use case? let's [talk](https://www.falkordb.com/get-demo/)**
+
+<br />
+
+# AI Agents with GraphRAG
+
+### Orchestrator
+The GraphRAG-SDK supports Knowledge Graph-based agents. Each agent is an expert in the data it has learned, and the orchestrator orchestrates the agents.

```

Comment:
It has learned?
There's no learning going on (I would say).
Rephrase to, Each agent is an expert in his domain.

Reply:
You right